autometa/common/external/bowtie.py

```diff
@@ -128,7 +128,8 @@ def align(db, sam, fwd_reads=None, rev_reads=None, se_reads=None, cpus=0, **kwar
     for flag, reads in zip(["-1", "-2", "-U"], [fwd_reads, rev_reads, se_reads]):
         if reads:
             reads_provided = True
-            params.append(f'{flag} {",".join(reads)}')
+            # params.append(f'{flag} {",".join(reads)}')
+            params.append(f"{flag} {reads}")
```

Comment:
This should be one or the other. No need to leave the commented out line

Reply:
I wasn't sure why you had put `{",".join(reads)}` I faced an error with this while running the test and removing `","` resolved it. However, I wasn't sire if this was the right approach as you must have put that for a reason.

Reply:
`reads` is a `List[str]`, so when emitting the message to the user I wanted to concatenate the list by comma delimiters.

Perhaps this is an issue with the quotes? 

You could try:

```python
params.append(f"{flag} {', '.join(reads)}")
```

or:

```python
reads_list = ", ".join(reads)}
params.append(f"{flag} {reads_list}")

```

Reply:
I am not sure why you want to concatenate the using a comma. `reads` tend to become a `str` when it is being looped through by the `for` loop. This is what I was experiencing when running it in an IDE:
```python
In [12]: fwd_reads = "path_to_fwd_reads"

In [13]: rev_reads = "path_to_rev_reads"

In [14]: se_reads="path_to_se_reads"

In [15]: for flag, reads in zip(["-1", "-2", "-U"], [fwd_reads, rev_reads, se_reads]):
    ...:     if reads:
    ...:         print(",".join(reads)+"\n")
    ...:         print(', '.join(reads)+"\n")
    ...: 
    ...: 
p,a,t,h,_,t,o,_,f,w,d,_,r,e,a,d,s

p, a, t, h, _, t, o, _, f, w, d, _, r, e, a, d, s

p,a,t,h,_,t,o,_,r,e,v,_,r,e,a,d,s

p, a, t, h, _, t, o, _, r, e, v, _, r, e, a, d, s

p,a,t,h,_,t,o,_,s,e,_,r,e,a,d,s

p, a, t, h, _, t, o, _, s, e, _, r, e, a, d, s
```